app/api/tip/route.ts

```diff
@@ -23,13 +18,24 @@ const getResponse = async (req: NextRequest): Promise<NextResponse> => {
     );
   }
 
-  if (tipperFid === recipientFid) {
+  if (tipperFid == recipientFid) {
     return NextResponse.json(
       { message: `Can not tip yourself`, tipRemaining: 0, totalTipOnPost: 0 },
       { status: 400 },
     );
   }
 
+  const { count } = await supabase
+    .from('tips_activity_log')
+    .select('post_hash')
+    .eq('sender', `${tipperFid}`)
+    .eq('post_hash', postHash);
+
+  if (count) return NextResponse.json(
+    { message: 'Cannot tip again.' },
+    { status: 400 },
+  );
```

Comment:
this will prevent someone leaving multiple tips on the same song right?
- why are we preventing this?